### PR TITLE
Pro 7465 sols redirect LS amend

### PR DESCRIPTION
**Pull request by sanjayparekh-hmcts**
Added dynamic select for LS amend

### CODE REVIEW

File: src/test/resources/solicitorWillTypeIntestacy.json [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+        "Country": "SolAddCo"
+      },
+      "solsSolicitorAppReference":"firmReference",
+      "solsSolicitorEmail":"rebecca.baker@hmcts.net",
+      "solsSolicitorPhoneNumber":null,
+      "deceasedForenames":"firstName",
+      "deceasedSurname":"lastName",
+      "deceasedDateOfDeath":"2018-01-01",
+      "deceasedDateOfBirth":"1990-01-01",
+      "willExists":"Yes",
```
Comment (SarahLittlejohn) [inline src/test/resources/solicitorWillTypeIntestacy.json:21]: Should "willExists" be "No"?

File: src/test/resources/solicitorWillTypeIntestacy.json [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      "registryLocation":"Birmingham",
+      "docsReceivedEmailNotificationRequested":false,
+      "grantIssuedEmailNotificationRequested":"No",
+      "primaryApplicantApplying":true,
+      "primaryApplicantFullName":"exfn exln",
+      "deceasedFullName":"firstName lastName",
+      "boExaminationChecklistQ1": "yes",
+      "boExaminationChecklistQ2": "yes",
+      "boExaminationChecklistRequestQA" : "yes",
+      "caseType" : "gop",
```
Comment (SarahLittlejohn) [inline src/test/resources/solicitorWillTypeIntestacy.json:140]: Should "caseType" be "intestacy"?

**Review by SarahLittlejohn — COMMENTED** [review]
Think a couple of unit tests need changes (solicitorWillTypeIntestacy.json and solicitorWillTypeAdmon.json although I am not familiar with an Admon case). But other than that all looks good.

File: src/test/resources/solicitorWillTypeIntestacy.json (hunk 1 above)
Comment (sanjayparekh-hmcts) [reply src/test/resources/solicitorWillTypeIntestacy.json:21]: for the purposes of the test it only uses solsWillType on the json, I just did copy-paste. It wont change the specific test output

File: src/test/resources/solicitorWillTypeIntestacy.json (hunk 2 above)
Comment (sanjayparekh-hmcts) [reply src/test/resources/solicitorWillTypeIntestacy.json:140]: for the purposes of the test it only uses solsWillType on the json, I just did copy-paste. It wont change the specific test output

**Review by SarahLittlejohn — APPROVED** [review]

**Pull request merged**